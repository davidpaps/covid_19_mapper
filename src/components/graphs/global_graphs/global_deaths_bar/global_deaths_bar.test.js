import * as React from "react";
import { shallow } from "enzyme";
import GlobalDeathsBar from "./global_deaths_bar";

describe("Global Deaths Bar", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<GlobalDeathsBar countries={[]} />)));

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(3);
  });

  it("should render a <Button />", () => {
    expect(wrapper.find("Button").length).toEqual(1);
  });

  it("should render a <HorizontalBar />", () => {
    expect(wrapper.find("HorizontalBar").length).toEqual(1);
  });

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
