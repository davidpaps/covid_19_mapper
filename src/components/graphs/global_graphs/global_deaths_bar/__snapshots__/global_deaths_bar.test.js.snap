// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Deaths Bar renders correctly 1`] = `ShallowWrapper {}`;
